ui/lib/core/addon/helpers/date-format.js

```diff
@@ -4,6 +4,12 @@ import { format, parseISO } from 'date-fns';
 export function dateFormat([date, style]) {
   // see format breaking in upgrade to date-fns 2.x https://github.com/date-fns/date-fns/blob/master/CHANGELOG.md#changed-5
   let number = typeof date === 'string' ? parseISO(date) : date;
+  if (!number) {
+    return;
+  }
+  if (number.toString().length === 10) {
```

Comment:
the timestamp coming in from pki expiration was a ten digit number, missing milliseconds. This catches those numbers.

Reply:
Can you add a test to cover this case?